classes/Carrier.php

```diff
@@ -1184,7 +1184,7 @@ public function getRangeSuffix($currency = null)
         }
         $suffix = Configuration::get('PS_WEIGHT_UNIT');
         if ($this->getShippingMethod() == Carrier::SHIPPING_METHOD_PRICE) {
-            $suffix = $currency->sign;
+            $suffix = $currency->symbol;
```

Comment:
I think it would be better to use the getter method here?

```suggestion
            $suffix = $currency->getSymbol();
```

Reply:
Three devs agreed with `->symbol` at #29605. I guess we need consistency :)